set.mm

```diff
@@ -24886,20 +24932,25 @@ of logic but rather presupposes the Axiom of Extensionality (see theorem
        of the definition checker.
 
        For a general discussion of the theory of classes, see
-       ~ mmset.html#class .  (Contributed by NM, 15-Sep-1993.) $)
+       ~ mmset.html#class .  (Contributed by NM, 15-Sep-1993.)  (Revised by BJ,
+       24-Jun-2019.) $)
     df-cleq $a |- ( A = B <-> A. x ( x e. A <-> x e. B ) ) $.
   $}
 
   ${
-    $d x A $.  $d x B $.  $d x y z $.
-    $( The same as ~ df-cleq with the hypothesis removed using the Axiom of
-       Extensionality ~ ax-ext .  (Contributed by NM, 15-Sep-1993.)  Revised to
-       make use of ~ axext3 instead of ~ ax-ext , so that ~ ax-9 will appear in
-       lists of axioms used by a proof, since ~ df-cleq implies ~ ax-9 by
-       theorem ~ bj-ax9 .  We may revisit this in the future.  (Revised by NM,
-       28-Oct-2021.)  (Proof modification is discouraged.) $)
+    $d x y z t u v A $.  $d x y z t u v B $.
+    $( The defining characterization of class equality.  It is proved, over
+       Tarski's FOL, from the axiom of (set) extensionality ( ~ ax-ext ) and
+       the definition of class equality ( ~ df-cleq ).  Its forward implication
+       is called "class extensionality".  Remark: the proof uses ~ axext4 to
+       prove also the hypothesis of ~ df-cleq that is a degenerate instance,
+       but it could be proved also from minimal propositional calculus and {
+       ~ ax-gen , ~ ax-4 }.  (Contributed by NM, 15-Sep-1993.)  (Revised by BJ,
```

Comment:
I don't think this is possible: the left hand side is `t = t` (equid) and the right hand side is ~ id + ~ ax-gen